django_app_server_db_server/deployment/roles/app_server/tasks/main.yml

```diff
@@ -65,32 +73,37 @@
   # - name: Upgrade conda
   # conda: name=conda state=latest executable=/opt/miniconda2/bin/conda
 
-- name: Make sure pillow installed
-  # conda: name=pillow state=latest executable=/opt/miniconda2/bin/conda
-  shell: /opt/miniconda2/bin/conda list | grep -q -i pillow || /opt/miniconda2/bin/conda install --yes -c conda-forge pillow
-  args:
-    creates: /opt/miniconda2/lib/python2.7/site-packages/PIL
-
 - name: Make sure that uwsgi is installed
   pip: name=uwsgi virtualenv=/opt/miniconda2 version=2.0.13
   #shell: /opt/miniconda2/bin/conda list | grep -q -i uwsgi || /opt/miniconda2/bin/conda install --yes -c travis uwsgi
   #args:
   #  creates: /opt/miniconda2/bin/uwsgi
 
-- name: Checkout eclaim source code
+- name: Make sure pillow installed for eclaim_revamp
+  # conda: name=pillow state=latest executable=/opt/miniconda2/bin/conda
+  shell: /opt/miniconda2/bin/conda list | grep -q -i pillow || /opt/miniconda2/bin/conda install --yes -c conda-forge pillow
+  args:
+    creates: /opt/miniconda2/lib/python2.7/site-packages/PIL
+  when: app_version == "eclaim_revamp"
+
+- name: Make sure pillow installed for eclaim
+  # conda: name=pillow state=latest executable=/opt/miniconda2/bin/conda
+  shell: /opt/miniconda2/bin/conda list | grep -q -i pillow || /opt/miniconda2/bin/conda install --yes -c ryan pillow
+  args:
+    creates: /opt/miniconda2/lib/python2.7/site-packages/PIL
+  when: app_version == "eclaim"
```

Comment:
This one you are sure ? I mean both the `Pillow` package for `eclaim_revamp` really cannot be used for `eclaim` is it ?